great_expectations_cloud/agent/agent.py

```diff
@@ -501,6 +505,11 @@ def _create_scheduled_job_and_set_started(
                     "organization_id": str(org_id),
                 },
             )
+        if response.ok is not True:
+            raise GXAgentError(  # noqa: TRY003 # one off error
+                f"RunScheduledCheckpointAction encountered an error while connecting to GX Cloud."
+                f"Correlation ID of event: {event_context.correlation_id}"
+            )
```

Comment:
I believe the response to a POST is a 201

Reply:
Yup this is a `201` : https://github.com/greatexpectationslabs/mercury/blob/b78b06b47c3a3707bab35096d70922ffafd4f6f9/services/ge_cloud/mercury/handlers/rest/agent_v0.py#L183